src/main/java/frc/robot/subsystems/SimulationSubsystem.java

```diff
@@ -46,6 +50,19 @@ public void simulationPeriodic() {
     //Update simYaw distance
     simYaw += chassisRotationSpeed *  Robot.kDefaultPeriod;
 
+    //Update simPitch
+    swerveX = swerveSubsystem.getPose().getX();
+    
+    if (swerveX >= 0 && swerveX <= FieldConstants.Community.chargingStationLength) {
+      simPitch = Units.degreesToRadians((swerveX * ((FieldConstants.Community.chargingStationAngle * 2) / FieldConstants.Community.chargingStationLength)) - FieldConstants.Community.chargingStationAngle);
```

Comment:
Split this up into multiple lines for readability

Reply:
Done